docs/beamrunner/beamrunner-5-ref.md

```diff
@@ -81,23 +83,68 @@ The Streams Runner package contains the following folders:
   </tbody>
 </table>
 
-## `STREAMING_ANALYTICS_SERVICE` context-specific pipeline options
+For the full list of pipeline options, enter  `--help=StreamsPipelineOptions` on the Beam application command line.
+
+#### `STREAMING_ANALYTICS_SERVICE` context-specific pipeline options
 
 | Parameter | Description | Default value |
 | --- | --- | --- |
 | `vcapServices` | The location of the Streaming Analytics VCAP file. This parameter is required when you use the `STREAMING_ANALYTICS_SERVICE` context type. This parameter can be omitted if the `$VCAP_SERVICES` environment variable is set to the path of the file. | [null] |
 | `serviceName` | The name of the Streaming Analytics service on IBM Cloud. This parameter is required when you use the   `STREAMING_ANALYTICS_SERVICE`  context type. | [null] |
 
-## `DISTRIBUTED` context-specific pipeline options
+#### `DISTRIBUTED` context-specific pipeline options
 
 | Parameter | Description | Default value |
 | --- | --- | --- |
 | `restUrl` | The URL for the REST API when you use the `DISTRIBUTED` context type. This parameter is required if you implement metrics in the application. | [null] |
 | `userName` | The user name for basic authentication with REST API when you use the `DISTRIBUTED` context type. | [null] |
 | `userPassword` | The user password for basic authentication for REST API when you use the `DISTRIBUTED` context type. | [null] |
 
-<br>
-For the full list of pipeline options, enter  `--help=StreamsPipelineOptions` on the Beam application command line.
+## Environment Variables
+These environment variables are not required for the Streams Runner to work; however, they can be used for convenience when launching your Beam application.

```

Comment:
Change
however, they can be used for convenience when launching your Beam application.
to
however, they can be used for convenience when you launch your Beam application.